### PR TITLE
Update 2fa toggle language

### PR DESCRIPTION
**Pull request by RasPhilCo**

### CODE REVIEW

File: lib/two_factor_toggle.js [hunk 1]
```diff
@@ -6,8 +6,8 @@ module.exports = function * (context, heroku, on) {
   let body = {password: password, two_factor_authentication: on}
   let account = yield heroku.patch('/account', {body: body})
   if (account.two_factor_authentication) {
-    cli.log('Two-factor authentication is enabled')
+    cli.log('Two-factor authentication has been enabled')
```
Comment (dickeyxxx) [inline lib/two_factor_toggle.js]: maybe a bit more concise:

```js
let enabled = account.two_factor_authentication ? 'enabled' : 'disabled'
cli.log(`Two-factor authentication has been ${enabled}`)
```

File: lib/two_factor_toggle.js [hunk 2]
```diff
@@ -6,8 +6,8 @@ module.exports = function * (context, heroku, on) {
...
   } else {
-    cli.log('Two-factor authentication is disabled')
+    cli.log('Two-factor authentication has been disabled')
```
Comment (RasPhilCo) [inline lib/two_factor_toggle.js]: FYI, we submit the patch request even if the account's 2fa is already disabled. Is it confusing if a user suspects the account has 2fa disabled but runs `disable` anyway and gets the `has been disabled` messaged when nothing actually changed?

Comment (dickeyxxx) [inline lib/two_factor_toggle.js]: yes, we should error that it's already disabled

File: lib/two_factor_toggle.js [hunk 3]
```diff
@@ -2,12 +2,15 @@
...
-    cli.log('Two-factor authentication is enabled')
+  let account = yield heroku.get('/account')
+  if ((on && account.two_factor_authentication) || (!on && !account.two_factor_authentication)) {
+    let enabled = account.two_factor_authentication ? 'enabled' : 'disabled'
+    throw new Error(`Two-factor authentication is already ${enabled}`)
   } else {
-    cli.log('Two-factor authentication is disabled')
+    let password = yield cli.prompt('Password', {hide: true})
+    let body = {password: password, two_factor_authentication: on}
+    let account = yield heroku.patch('/account', {body: body})
```
Comment (dickeyxxx) [inline lib/two_factor_toggle.js]: minor improvement:

```js
let body = {password, two_factor_authentication: on}
let account = yield heroku.patch('/account', {body})
```

**Pull request merged**